src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue

```diff
@@ -215,10 +253,37 @@ if (choiceItem.value) {
   $this: &;
   $element: multiple-answer-item;
 
+  &--has-info {
+    #{$this}__label {
+      &::after {
+        @include mixins.use-logical($element + "-suffix", margin, 0 0 0 var(--space-12));
+
+        width: var(--icon-size, var(--icon-width, 1.5rem));
+        height: var(--icon-size, var(--icon-height, 1.5rem));
+        flex: none;
+        content: "";
+      }
+    }
+  }
+
   &__label {
+    position: relative;
     display: flex;
     align-items: flex-start;
     justify-content: space-between;
   }
+
+  &__suffix {
+    @include mixins.override-logical(list-item-suffix, null, margin, 0);
+
+    position: absolute;
+    inset-block-start: 0;
+    inset-inline-end: 0;
+    @include mixins.use-logical("list-item-content", padding, var(--space-12) var(--space-20));
```

Comment:
to keep relation between `ui-list-item-content` and spacings of this absolute positioning element. User can change only `ui-list-item-content-paddin` and doesn't have to worry to adjusting spaces of this suffix element